gax-java/gax-httpjson/src/test/java/com/google/api/gax/httpjson/HttpJsonDirectServerStreamingCallableTest.java

```diff
@@ -68,6 +68,7 @@
 
 @RunWith(JUnit4.class)
 public class HttpJsonDirectServerStreamingCallableTest {
+
```

Comment:
remove

Reply:
> Let's run it to see any failures.

https://github.com/googleapis/gapic-generator-java/actions/runs/4010427729/jobs/6887076787

This is the commit before the swap to scheduling via cron.